src/main/kotlin/blackjack/domain/Player.kt

```diff
@@ -5,14 +5,30 @@ import blackjack.domain.enums.Condition
 class Player(
     name: String,
     cards: Cards,
-    condition: Condition = Condition.PLAY
+    condition: Condition = Condition.PLAY,
+    val betAmount: Double,
 ) : Participant(name, cards, condition) {
 
+    var money: Double = betAmount
+        private set
+
     fun currentCondition(): Condition {
         return this.condition
     }
```

Comment:
상태를 나타낸다면 프로퍼티로 제공하면 어떨까요?
Participant에서 아래와 같이 프로퍼티에 getter를 붙이면 항상 현재 상태를 반환하게 만들 수 있습니다🙂
```suggestion
    // Participant.kt
    var condition: Condition get() = this.condition
        protected set
```